### PR TITLE
feat(connect-data): Getting connection data

### LINKED ISSUE #357: Fetch DB connection details from GraphQL server to display at `/connect`
Since #273 has merged, we should finish out the `/connect` page to display to the user how reach the DB.

- [ ] Implement correct URL components (except for password) fetched from GraphQL API (make sure to account for loading states)
- [ ] Implement copy buttons

### PR DESCRIPTION
**Pull request by german-mergestat**
Getting connection data from database. (Resolves #357)

**Note:** I will avoid to keep implementing unit testing due application is a little changeable for now. I understand we are gonna focus more in unit testing for v2 any way. 

connection-data

### CODE REVIEW

File: ui/src/pages/connect/index.tsx [hunk 1]
```diff
@@ -19,98 +30,104 @@ const Connect: NextPage = () => {
...
-                  <p className="text-gray-500">This is some helptext with a short description to explain how to use this connect page.</p>
-                </div>
+        {loading
+          ? <Loading />
+          : <div className="flex-1 items-center p-8 overflow-auto">
+            <div className="sm_mx-auto sm_max-w-5xl">
+              <Panel className="shadow-sm rounded-md">
+                <Panel.Body className="p-8 overflow-hidden">
+                  <div className="mb-6">
+                    <h3 className="t-h3">Connection details</h3>
```
Comment (patrickdevivo) [inline ui/src/pages/connect/index.tsx:40]:
```suggestion
                    <h3 className="t-h3">Connect to PostgreSQL Directly</h3>
```

File: ui/src/pages/connect/index.tsx [hunk 2]
```diff
@@ -19,98 +30,104 @@ const Connect: NextPage = () => {
...
+                    <p className="text-gray-500">This is some helptext with a short description to explain how to use this connect page.</p>
```
Comment (patrickdevivo) [inline ui/src/pages/connect/index.tsx:41]:
```suggestion
                    <p className="text-gray-500">Connect directly to the PostgreSQL instance MergeStat syncs data to.</p>
```

**Review by patrickdevivo — CHANGES_REQUESTED** [review]

File: ui/src/pages/connect/index.tsx (hunk 1 above)
Comment (german-mergestat) [reply ui/src/pages/connect/index.tsx]: Done [2313b24](https://github.com/mergestat/fuse/pull/389/commits/2313b24774c02ea63c7ac857dc516c405b237d97)

File: ui/src/pages/connect/index.tsx (hunk 2 above)
Comment (german-mergestat) [reply ui/src/pages/connect/index.tsx]: Done [2313b24](https://github.com/mergestat/fuse/pull/389/commits/2313b24774c02ea63c7ac857dc516c405b237d97)

**Review by patrickdevivo — APPROVED** [review]

**Pull request merged**